TS-Example/index.ts

```diff
@@ -219,6 +242,52 @@ const activateValidator = async (
   }
 };
 
+/**
+ * Returns incentives data for a specific address
+ * @param address The Ethereum address to check for incentives
+ * @returns The incentives data including amount, index, merkle proof, and contract address
+ */
+const getObolIncentivesByAddress = async (address: string): Promise<Incentives> => {
+  try {
+    //const client = await obolClientًWithoutProvider();
+    const incentivesData = await client.incentives.getIncentivesByAddress(address);
+    return incentivesData;
+  } catch (err) {
+    console.log(err, "err");
+  }
+};
+
+/**
+ * Checks if incentives have already been claimed for a specific index
+ * @param contractAddress The merkle distributor contract address
+ * @param index The index in the merkle tree
+ * @returns Boolean indicating if the incentives have been claimed
+ */
+const isObolIncentivesClaimed = async (contractAddress: string, index: number): Promise<boolean> => {
+  try {
+    //const client = await obolClientًWithProviderOnly();
+    const claimed = await client.incentives.isClaimed(contractAddress, index);
+    return claimed;
+  } catch (err) {
+    console.log(err, "err");
+  }
+};
+
+/**
+ * Claims incentives for a specific address
+ * @param address The Ethereum address for which to claim incentives
+ * @returns Object containing txHash if successful or alreadyClaimed: true if already claimed
+ */
+const claimObolIncentives = async (address: string): Promise<{ txHash: string } | { alreadyClaimed: boolean }> => {
+  try {
+    //const client = await obolClient();
+    const claimResult = await client.incentives.claimIncentives(address);
+    return claimResult;
+  } catch (err) {
+    console.log(err, "err");
+  }
+};
```

Comment:
oh sorry missed this before, but would a better behaviour potentially be that claimIncentives returns:
```
{
  txHash: Nullable(string),
  alreadyClaimed: boolean
}
```